bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -188,18 +122,6 @@
 			<channel-group id="forecastHours10" typeId="oneCallHourly"/>
 			<channel-group id="forecastHours11" typeId="oneCallHourly"/>
 			<channel-group id="forecastHours12" typeId="oneCallHourly"/>
-			<channel-group id="forecastHours13" typeId="oneCallHourly"/>
```

Comment:
Same question here. Default is now 24 hours, and up to 48 is supported I think? It seems to be 48 in handler code and 120 in config definition:
https://github.com/openhab/openhab-addons/blob/5f805755fc60347ab7a548b6a11fea7ea1efe2b3/bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java#L110-L115

https://github.com/openhab/openhab-addons/blob/5f805755fc60347ab7a548b6a11fea7ea1efe2b3/bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/config/config.xml#L88-L92